clippy_lints/src/literal_representation.rs

```diff
@@ -504,3 +551,11 @@ impl LiteralRepresentation {
         Ok(())
     }
 }
+
+fn is_mistyped_suffix(suffix: &str) -> bool {
+	["_8", "_16", "_32", "_64", "_128"].contains(&suffix)
```

Comment:
NIT: indentation, please use 4 spaces.

Reply:
I think calling `cargo fmt --all` should fix it

Reply:
Yeah probably, but please make sure to not add unrelated formatted files.

Reply:
I'll check the files I've touched to make sure they're all properly transformed. Unsure why it happened, my editor should have read the `.editorconfig` and used the settings there.